src/CustomTheme.tsx

```diff
@@ -0,0 +1,22 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+import { grey, teal } from 'material-ui-colors';
```

Comment:
I believe, it should be like that, it throws an error otherwise.
```suggestion
import { grey, teal } from '@material-ui/core/colors';
```